spec/javascript/components/ShowLocationTile.js

```diff
@@ -0,0 +1,33 @@
+import ShowTile from '../../../app/javascript/react/components/ShowTile'
```

Comment:
delete this it is handled by Pablos PR that was merged.

Reply:
the whole file

Reply:
?